src/request.rs

```diff
@@ -277,6 +277,47 @@ impl RequestBuilder {
         Ok(self)
     }
 
+    /// Sends a multipart/formdata body.
+    ///
+    /// ```no_run
```

Comment:
I think for that to work I will need to create a `MultipartError` and add that to `error.rs`, yes? Otherwise the errors of `to_multipart` and `MultipartField::file` will not get converted to `reqwest::Error`.